### PR TITLE
Add command to open screenshot settings

### PR DESCRIPTION
**Pull request by xavier630**
This is an important UI on mac for taking screenshots as it's where you decide where to save it, whether it will be a video, if a timer should be used, etc.

I'm open to ideas for the command naming, the naming I went for matches the pattern in the file but could be more intuitive.

### CODE REVIEW

**Review by splondike — APPROVED** [review]
Looks good to merge after somebody else using OSX verifies.

**Comment by nriley** [comment]
I don't think this is a particularly good idea as the keyboard shortcuts for taking screenshots on macOS are customizable. Potentially wrapping the screencapture command line utility would be a better idea.

**Comment by nriley** [comment]
I'm also happy just to submit a PR that uses screencapture if @xavier630 doesn't want to. It wouldn't be a lot of work.

**Comment by xavier630** [comment]
Hey team,

Agreed with Splondike that I'd rather merge this and have a separate piece of work to convert to CLI functionality.

I'd rather not block on it, especially since this follows the existing pattern in the file: which already assumes default shortcuts are used:
image

File: code/screenshot.py
Comment (rntz) [inline code/screenshot.py:50]: Please add an `else` branch that raises an error if we're not on mac.

**Review by rntz — CHANGES_REQUESTED** [review]

File: code/screenshot.py
Comment (xavier630) [reply code/screenshot.py:50]: @rntz I've now added this. I can't test it due to my OS, but the code is pretty basic.

**Comment by splondike** [comment]
Merging this in. If somebody wants to re-implement using the CLI, the coast is now clear :).

**Pull request merged**
